.github/workflows/ci.yml

```diff
@@ -42,12 +42,12 @@ jobs:
         java-package: jre
     - run: npm install
     - name: Use Dumper
-      run: |
-        mkdir output
-        node bin/dumpPackets -v ${{ matrix.mcVersionIndex }} -o output/${{ matrix.mcVersionIndex }}
-        
+      run: node bin/dumpPackets -v ${{ matrix.mcVersionIndex }}
```

Comment:
this is incorrect, why did you remove the `output/${{ matrix.mcVersionIndex }}` ?

Reply:
must've merged wrong